### PR TITLE
Add Sentry fingerprints for postgres errors

### LINKED ISSUE #3894: Help Sentry to tell postgres errors apart
https://sentry.prod.mozaws.net/operations/taskcluster-firefox-ci/issues/9746308/

Clicking "older" a few times shows a whole menagerie of error messages lumped into this sentry report.  Which message it chooses to quote in the alert email seems arbitrary, or at least not "the message of the error that just occurred".

There are ways to add context to Sentry that it will use to decide if an error matches an existing issue.  And, postgres provides a really nicely detailed `code` property to all of its errors.  Maybe those two can be glued together well enough to improve this situation.

### PR DESCRIPTION
**Pull request by djmitche**
Brian, any chance I could use your dev env to test this somehow?

Github Bug/Issue: Fixes #3894

### CODE REVIEW

**Comment by djmitche** [comment]
image

looks good!

File: libraries/monitor/src/plugins/sentry.js [hunk 1]
```diff
@@ -27,7 +27,7 @@ class SentryReporter {
   }
 
   report(error, level, extra) {
-    Sentry.configureScope(scope => {
+    Sentry.withScope(scope => {
```
Comment (imbstack) [inline libraries/monitor/src/plugins/sentry.js:30]: Ah this probably explains a lot of weirdness in our reports

**Review by imbstack — APPROVED** [review]
very nice

File: libraries/monitor/src/plugins/sentry.js (hunk 1 above)
Comment (djmitche) [reply libraries/monitor/src/plugins/sentry.js:30]: Especially the weird breadcrumbs :)

**Pull request merged**